contributing.md

```diff
@@ -0,0 +1,141 @@
+# Contributing
+
+> You want to contribute to the website? Welcome!
+> You will find here resources and advices that could help you.
+
+If you have any questions, you can ask them in #website-dev channel in our Discord.
+
+ * [Preparing the development environment](#preparing-the-development-environment)
+ * [Guidelines](#guidelines-1)
+ * [Localization](#localization)
+
+<!-- /code_chunk_output -->
+
+
+## Preparing the development environment
+### Fork the project
+* Fork the repo.
+* **Always create a new branch before working on your fork.**
+* You can now improve the website.
+
+> You can use a git client like [GitHub Desktop](https://desktop.github.com/) to manage your local repository.
+
+When your done submit a pull request.
```

Comment:
Be clear for people who haven't used Git before
```suggestion
When your done submit a pull request. This can be done by navigating to the main page of the [website git repository](https://github.com/Paperback-iOS/website) and selecting 'pull request' at the top.  You may have to click the button titled **compare across forks**. Ensure that the base repository belongs to Paperback-iOS, whereas the head is your fork, and your branch. ``

Reply:
Nice!
Done in ea456ca.